documentation/docs/verifiable_credentials/create.mdx

```diff
@@ -12,15 +12,67 @@ keywords:
 import create_vc_rs from  '!!raw-loader!./../../../examples/account/create_vc.rs';
 import CodeSnippet from '../../src/components/CodeSnippetComponent'
 
+A verifiable credential can represent all information that a phisical [credential](https://www.w3.org/TR/vc-data-model/#dfn-credential) represents (i.e a set of one or more claims made by an issuer. However, by allowing other parties to cryptographically verify the authorship of the credential, a VC can be seen as more tamper-evident and more trustworthy than their physical counterparts. 
```

Comment:
Fix typos. Suggest examples. Introduce VC abbreviation. This also overlaps with the introductory article, suggest changing link target to that.

```suggestion
A [verifiable credential (VC)](./verifiable_presentations.mdx) can represent all information that a physical credential represents, such as as a passport or university degree. However, by allowing other parties to cryptographically verify the authorship and integrity of the claims, verifiable credentials can be seen as more tamper-evident and more trustworthy than their physical counterparts.
```